.go-version

```diff
@@ -1 +1 @@
-1.13.9
+1.14.2
```

Comment:
Any moves to a new version of go deserve their own PR. That being said, we intentionally reverted back to go1.13 from go1.14 due to the preemptive scheduler ticking a bug in the linux kernel. We shouldn't move forward until we have determined that the risk of hitting an unpatched kernel is low.